integration-commons/pom.xml

```diff
@@ -55,5 +55,9 @@
             <groupId>commons-codec</groupId>
             <artifactId>commons-codec</artifactId>
         </dependency>
+        <dependency>
```

Comment:
This dependency is only required for integrations that need to authenticate using OAuth protocol. So, it shouldn't be part of the commons module.